flask_webpackext/ext.py

```diff
@@ -53,6 +54,10 @@ def init_config(self, app):
             "WEBPACKEXT_MANIFEST_LOADER",
             JinjaManifestLoader,
         )
+        app.config.setdefault(
+            "FLASK_WEBPACKEXT_NPM_PKG_CLS",
```

Comment:
maybe without the `FLASK_` prefix. no other configuration variable in flask-webpackext has the `FLASK_` as prefix

Reply:
right, good catch